e2e/steps_file.js

```diff
@@ -31,5 +31,15 @@ module.exports = function () {
       this.click(link);
       this.seeCurrentUrlEquals(urlNavigatedTo);
     },
+
+    seeDocument(title, name, status, reason = '') {
```

Comment:
if you put name = ' ' in the constructor, then in uploadDocuments_test it saves you passing in an empty string